features-json/template-literals.json

```diff
@@ -200,9 +202,9 @@
       "10.1":"y",
       "11":"y",
       "11.1":"y",
-      "12":"y",
-      "12.1":"y",
-      "TP":"y"
+      "12":"a",
+      "12.1":"a",
```

Comment:
it's fixed in 12.1. PR to compat-table is here https://github.com/kangax/compat-table/pull/1427

Reply:
Actually bug is still there so LGTM